app/classifier/index.cjsx

```diff
@@ -445,6 +477,12 @@ Classifier = React.createClass
   renderBackButtonWarning: ->
     <p className="back-button-warning" >Going back will clear your work for the current task.</p>
 
+  subjectIsGravitySpyGoldStandard: ->
+    if @props.workflow.configuration?.gravity_spy_gold_standard and @props.subject.metadata?['#Type'] is 'Gold'
```

Comment:
This line can be just `@props.workflow.configuration?.gravity_spy_gold_standard and @props.subject.metadata?['#Type'] is 'Gold'` and either true or false will be returned.